### PR TITLE
fix(service overview): remove multiple errors

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Service overview: Added missing translations and fixed duplicate error
Package: Updated shared components dependency and removed duplicate log

## Why

Service overview: since only one error should be displayed

## Issue

Ref: CPLP-3545, CPLP-3473

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -104,7 +109,6 @@
 - Release Process (Apps & Services)
   - information message "no {object} available" getting displayed while content is available
 - App Release Process
-  - fixed automatic upscrolling in form pages
```
Comment (evegufy) [inline CHANGELOG.md:107]: did you intend to change the changelog from a already released version?

**Review by evegufy — CHANGES_REQUESTED** [review]
Please change to a more descriptive pr title and remove all non relevant options in the checklist of the pr description.

File: CHANGELOG.md (hunk 1 above)
Comment (lavanya-bmw) [reply CHANGELOG.md:107]: @evelyn It was added twice(line 107 and line113), so removed duplicate log

**Review by nidhigarg-bmw — APPROVED** [review]
lgtm

**Review by jjeroch — CHANGES_REQUESTED** [review]
@lavanya-bmw  as mentioned in this pr https://github.com/eclipse-tractusx/portal-frontend/pull/404 the changeLog was incorrectly updated. Now looking at this I can still see here the incorrect changeLog as well as on dev. Was this not fixed with Evelyn jointly? This needs to get adjusted before we can move ahead. Changing a previous ChangeLog is a major issue...also please make sure that review comments are completely handled/managed.

**Review by oyo — APPROVED** [review]

**Comment by lavanya-bmw** [comment]
> @lavanya-bmw as mentioned in this pr #404 the changeLog was incorrectly updated. Now looking at this I can still see here the incorrect changeLog as well as on dev. Was this not fixed with Evelyn jointly? This needs to get adjusted before we can move ahead. Changing a previous ChangeLog is a major issue...also please make sure that review comments are completely handled/managed.

@jjeroch I have reverted the changelog from previous release

File: CHANGELOG.md (hunk 1 above)
Comment (lavanya-bmw) [reply CHANGELOG.md]: > @evelyn It was added twice(line 107 and line113), so removed duplicate log

@evegufy  I have reverted the changelog from previous release

**Pull request merged**